Tools/Taxonomy/Cwe/CweTaxonomyGenerator.cs

```diff
@@ -0,0 +1,365 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.Serialization;
+
+using CsvHelper;
+
+using Microsoft.CodeAnalysis.Sarif;
+
+using Newtonsoft.Json;
+
+using Taxonomy.Common;
+
+namespace Taxonomy.Cwe
+{
+    public class CweTaxonomyGenerator : TaxonomyGenerator
+    {
+        public bool SaveCsvToSarif(string filePath, string version, string releaseDateUtc)
```

Comment:
>SaveCsvToSarif [](http://example.com/codeflow?start=20&length=14)

are u using this?

Reply:
no this was the original method for generate from CSV, I added a method for XML. we can leave it for now and if later we are sure we never need it we can delete it later.